docs/user-guide/cli-install-configure-zosmf.md

```diff
@@ -1,6 +1,8 @@
 # Configuring z/OSMF
 
-Complete the following IBM z/OSMF configuration tasks for your Zowe CLI implementation.
+For client-side components to communicate with the mainframe, z/OSMF requires configuration to make this happen.
+
+Complete the following IBM z/OSMF configuration tasks for the implementation of Zowe CLI, Zowe Explorer for Visual Studio Code, or Zowe IntelliJ Plug-in. **[is this intro correct? are any edits needed for the rest of this page?]**
```

Comment:
@adam-wolfe please respond

Reply:
I would add a "the" before "Zowe IntelliJ Plug-in". We normally call it "Zowe Explorer for IntelliJ", but on Zowe.org all I see is "Zowe IntelliJ Plug-in". I'm not sure what to do about that, might be a good question for TSC.